core/vtk/ttkPersistenceDiagramDictionary/ttkPersistenceDiagramDictionary.cpp

```diff
@@ -0,0 +1,221 @@
+#include <ttkMacros.h>
+#include <ttkPersistenceDiagramDictionary.h>
+#include <ttkPersistenceDiagramUtils.h>
+#include <ttkUtils.h>
+
+#include <vtkAlgorithm.h>
+#include <vtkCellData.h>
+#include <vtkCharArray.h>
+#include <vtkDataArray.h>
+#include <vtkDataSet.h>
+#include <vtkDoubleArray.h>
+#include <vtkFiltersCoreModule.h>
+#include <vtkFloatArray.h>
+#include <vtkIntArray.h>
```

Comment:
that include does not seem necessary.
please remove all unnecessary includes (i.e. delete the line and see if it still builds)